doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1039,42 +1066,22 @@ local definitions) with:
 
    :tacn:`fun` is a :token:`binder_tactic`.
 
-A tactic can return a non-tactic value only by ending the expression
-with the idiom :n:`let … in @ident`.
-
-.. todo Another problem with the coqtop directive example:: Returning a value from a function
-
-   For example:
-
-   The wrapping in `let ... lazymatch` is essential.  It works because :tacn:`lazymatch`
-   executes its first match during the evaluation phase.
-
-   .. coqtop:: reset none
-
-      Goal True.
-
-   .. coqtop:: all
-
-      Ltac x := let n := numgoals in
-                let b := lazymatch True with | _ => idtac "run" end in
-                n.
-      let m := x in idtac m.
-
-   While this doesn't return the desired type:
-
-   .. coqtop:: all
-
-      Ltac x2 := idtac "x"; numgoals.
-      let y := x2 in idtac y.
+Functions can return values of any type.
 
 A function application is an expression of the form:
 
-.. tacn:: @qualid {* @tactic_arg }
+.. tacn:: @qualid {+ @tactic_arg }
```

Comment:
Note that the grammar table still uses the `*` form; is this mismatch intentional?

Reply:
Oversight; the grammar-editing is not entirely foolproof (> 2000 lines of edit instructions!)